### PR TITLE
Public announcement April - 2024

### PR DESCRIPTION
**Pull request by anjai94**
Following advisory will be announced via this PR. 
- WSO2-2022-1754
- WSO2-2022-1988
- WSO2-2022-2049
- WSO2-2022-2165
- WSO2-2022-2203
- WSO2-2022-2208
- WSO2-2022-2254
- WSO2-2023-2075
- WSO2-2023-2308
- WSO2-2023-2577
- WSO2-2023-2626
- WSO2-2023-2831
- WSO2-2023-2935

### CODE REVIEW

**Review by 1uffyD9 — CHANGES_REQUESTED** [review]
Added few comments. 

Additionally for following advisories, public PRs are missing,

- WSO2-2022-1754
- WSO2-2022-1988

Also, I noted one more different content in the `WSO2-2022-2203` advisory. Could you please check the content against the PMT.

File: en/docs/security-announcements/security-advisories/2024/WSO2-2022-2254.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+category: security-announcements
+published: May 31, 2024
+version: 1.0.0
+severity: Medium
+cvss: "6.1 (CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N)"
+---
+
+# SECURITY ADVISORY WSO2-2022-2254
+
+<p class="doc-info">Published: April 20, 2024</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2022-2254.md:12]:
```suggestion
<p class="doc-info">Published: May 31, 2024</p>
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2022-2208.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+category: security-announcements
+published: May 31, 2024
+version: 1.0.0
+severity: High
+cvss: "8.9 (CVSS:3.1/AV:N/AC:H/PR:N/UI:N/S:C/C:H/I:H/A:L)"
+---
+
+# SECURITY ADVISORY WSO2-2022-2208
+
+<p class="doc-info">Published: April 20, 2024</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2022-2208.md:12]:
```suggestion
<p class="doc-info">Published: May 31, 2024</p>
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2022-2203.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+category: security-announcements
+published: May 31, 2024
+version: 1.0.0
+severity: Medium
+cvss: "4.8 (CVSS:3.1/AV:N/AC:L/PR:H/UI:R/S:C/C:L/I:L/A:N)"
+---
+
+# SECURITY ADVISORY WSO2-2022-2203
+
+<p class="doc-info">Published: April 20, 2024</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2022-2203.md:12]:
```suggestion
<p class="doc-info">Published: May 31, 2024</p>
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2022-2165.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+category: security-announcements
+published: May 31, 2024
+version: 1.0.0
+severity: High
+cvss: "7.5 (CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:H/I:N/A:N)"
+---
+
+# Security Advisory WSO2-2022-2165
+
+<p class="doc-info">Published: October 28, 2023</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2022-2165.md:12]:
```suggestion
<p class="doc-info">Published: May 31, 2024</p>
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2022-2049.md [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+category: security-announcements
+published: May 31, 2024
+version: 1.0.0
+severity: Medium
+cvss: "6.1 (CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N)"
+---
+
+# SECURITY ADVISORY WSO2-2022-2049
+
+<p class="doc-info">Published: April 20, 2024</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2022-2049.md:12]:
```suggestion
<p class="doc-info">Published: May 31, 2024</p>
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2022-1988.md [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+category: security-announcements
+published: May 31, 2024
+version: 1.0.0
+severity: Medium
+cvss: "5.4 (CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:U/C:L/I:L/A:N)"
+---
+
+# SECURITY ADVISORY WSO2-2022-1988
+
+<p class="doc-info">Published: April 20, 2024</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2022-1988.md:12]:
```suggestion
<p class="doc-info">Published: May 31, 2024</p>
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2023-2075.md [hunk 7]
```diff
@@ -0,0 +1,48 @@
...
+category: security-announcements
+published: May 31, 2024
+version: 1.0.0
+severity: Medium
+cvss: "4.3 (CVSS:3.1/AV:A/AC:L/PR:H/UI:R/S:C/C:L/I:L/A:N)"
+---
+
+# SECURITY ADVISORY WSO2-2023-2075
+
+<p class="doc-info">Published: April 20, 2024</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2023-2075.md:12]:
```suggestion
<p class="doc-info">Published: May 31, 2024</p>
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2023-2308.md [hunk 8]
```diff
@@ -0,0 +1,46 @@
+---
+title: Security Advisory WSO2-2023-2308
+category: security-announcements
+published: May 31, 2024
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2023-2308.md:4]:
```suggestion
published: May 31, 2024
severity: N/A
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2023-2308.md [hunk 9]
```diff
@@ -0,0 +1,46 @@
...
+version: 1.0.0
+---
+
+# SECURITY ADVISORY WSO2-2023-2308
+
+<p class="doc-info">Published: April 20, 2024</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2023-2308.md:10]:
```suggestion
<p class="doc-info">Published: April 20, 2024</p>
<p class="doc-info">Severity: N/A</p>
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2023-2577.md [hunk 10]
```diff
@@ -0,0 +1,47 @@
...
+category: security-announcements
+published: May 31, 2024
+version: 1.0.0
+severity: Medium
+cvss: "5.3 (CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:L/I:N/A:N)"
+---
+
+# SECURITY ADVISORY WSO2-2023-2577
+
+<p class="doc-info">Published: April 20, 2024</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2023-2577.md:12]:
```suggestion
<p class="doc-info">Published: May 31, 2024</p>
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2023-2626.md [hunk 11]
```diff
@@ -0,0 +1,66 @@
...
+category: security-announcements
+published: May 31, 2024
+version: 1.0.0
+severity: Medium
+cvss: "6.1 (CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N)"
+---
+
+# SECURITY ADVISORY WSO2-2023-2626
+
+<p class="doc-info">Published: April 20, 2024</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2023-2626.md:12]:
```suggestion
<p class="doc-info">Published: May 31, 2024</p>
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2023-2831.md [hunk 12]
```diff
@@ -0,0 +1,38 @@
...
+category: security-announcements
+published: May 31, 2024
+version: 1.0.0
+severity: Medium
+cvss: "6.1 (CVSS:3.1/AV:N/AC:L/PR:N/UI:R/S:C/C:L/I:L/A:N)"
+---
+
+# SECURITY ADVISORY WSO2-2023-2831
+
+<p class="doc-info">Published: April 20, 2024</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2023-2831.md:12]:
```suggestion
<p class="doc-info">Published: May 31, 2024</p>
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2023-2935.md [hunk 13]
```diff
@@ -0,0 +1,42 @@
...
+category: security-announcements
+published: May 31, 2024
+version: 1.0.0
+severity: Medium
+cvss: "4.7 (CVSS:3.1/AV:N/AC:H/PR:N/UI:R/S:C/C:L/I:L/A:N)"
+---
+
+# SECURITY ADVISORY WSO2-2023-2935
+
+<p class="doc-info">Published: April 20, 2024</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2023-2935.md:12]:
```suggestion
<p class="doc-info">Published: May 31, 2024</p>
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2023-2938.md [hunk 14]
```diff
@@ -0,0 +1,97 @@
...
+category: security-announcements
+published: May 31, 2024
+version: 1.0.0
+severity: High
+cvss: 7.2 (CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:H/I:H/A:H)
+---
+
+# SECURITY ADVISORY WSO2-2023-2938
+
+<p class="doc-info">Published: April 20, 2024</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2023-2938.md:12]:
```suggestion
<p class="doc-info">Published: May 31, 2024</p>
```

File: en/docs/security-announcements/security-advisories/2024/WSO2-2023-2988.md [hunk 15]
```diff
@@ -0,0 +1,48 @@
...
+category: security-announcements
+published: May 31, 2024
+version: 1.0.0
+severity: High
+cvss: "7.2 (CVSS:3.1/AV:N/AC:L/PR:H/UI:N/S:U/C:H/I:H/A:H)"
+---
+
+# SECURITY ADVISORY WSO2-2023-2988
+
+<p class="doc-info">Published: April 20, 2024</p>
```
Comment (1uffyD9) [inline en/docs/security-announcements/security-advisories/2024/WSO2-2023-2988.md:12]:
```suggestion
<p class="doc-info">Published: May 31, 2024</p>
```

**Review by 1uffyD9 — CHANGES_REQUESTED** [review]
Added few suggestions

**Review by 1uffyD9 — APPROVED** [review]
+1 from my end.

**Review by ayomawdb — APPROVED** [review]

**Pull request merged**